packages/react-examples/src/azure-themes/stories/components/ProgressIndicator.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
```

Comment:
Added progressindicator control, as it was using another semanticcolor I wanted to check on